gemrb/core/MurmurHash.h

```diff
@@ -0,0 +1,55 @@
+/* GemRB - Infinity Engine Emulator
+* Copyright (C) 2024 The GemRB Project
+*
+* This program is free software; you can redistribute it and/or
+* modify it under the terms of the GNU General Public License
+* as published by the Free Software Foundation; either version 2
+* of the License, or (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with this program; if not, write to the Free Software
+* Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+*
+*/
+
+#ifndef H_GEMRB_MURMUR_HASH
+#define H_GEMRB_MURMUR_HASH
```

Comment:
What's the benefit over using our existing hashing capabilities, std?

Reply:
std needs a specialization for this case, as far as I see, and thus some implementation. And since I'm no hashing export, this one is safe and fast to use for streaming 4-bytes.

Reply:
Did you forget to commit something? I don't see it actually being used anywhere.